### DISCUSSION TITLE
Can not ignore KaiOs

### DISCUSSION THREAD

**Discussion by jimblue** [browserslist/browserslist#779] [General]
Hi there,

I'm trying to ignore KaiOs from my Browserlist config.

**Here is my old working config:**

```json
"browserslist": [
    "> 0.5%",
    "last 2 versions",
    "Firefox ESR",
    "not ie <= 11",
    "not op_mini all",
    "not dead"
  ],
```

**And the one one which is not working:**

```json
"browserslist": [
    "> 0.5%",
    "last 2 versions",
    "Firefox ESR",
    "not ie <= 11",
    "not kaios all",
    "not op_mini all",
    "not dead"
  ],
```

**I'm getting the following error:**

> BrowserslistError: Unknown browser query `not kaios all`. Maybe you are using old Browserslist or made typo in query.

What I'm doing wrong ? 🤔 

Thanks for your help 🙏🏼

**Comment by ai** [comment]
There is no query `not X all`. It works for Opera Mini because it has version as a `all' string.

You need to use `not kaios > 0` to exclude all users.

But I can't recommend it because you are excluding the whole Africa (this browser is very popular there).

Check out our Browserslist config linter results:

https://browsersl.ist/#q=%3E+0.5%25%2C%0Alast+2+versions%2C%0AFirefox+ESR%2C%0Anot+ie+%3C%3D+11%2C%0Anot+kaios+%3E+0%2C%0Anot+op_mini+all%2C%0Anot+dead

**Reply by jimblue** [reply]
Thank you so much for the explanation and the advice Andrey ! :)
Gona take it into consideration.
